group_project_v2/project_api/dtos.py

```diff
@@ -24,7 +34,7 @@ def __init__(self, **kwargs):
     def full_name(self):
         if self._full_name:
             return self._full_name
-        return u" ".join([unicode(part) for part in (self.first_name, self.last_name) if part is not None])
+        return super(UserDetails, self).full_name
```

Comment:
Nit: you might as well move the `_full_name` attribute into the base class.  (It doesn't matter, really, but I like to keep layers of indirection to a minimum.)